### DISCUSSION TITLE
Add additional column in finder

### DISCUSSION THREAD

**Discussion by heijligers** [sbarex/MediaInfo#19] [General]
Can I use this tool to add the aspect ratio of movie files to the macos finder display? So outside the context menu?

**Comment by sbarex** [comment]
No, Apple doesn't allow this behavior. But I have some ideas... ;-)

**Comment by heijligers** [comment]
Thanks for clearing that up! Would it be possible to use MediaInfo to batch
create custom tags based on the aspect ratio for selected files?


On Wed, 1 Mar 2023 at 08:53, sbarex ***@***.***> wrote:

> No, Apple doesn't allow this behavior.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/sbarex/MediaInfo/discussions/19#discussioncomment-5163851>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAX7BVYPY6SHYNCSWREZMPDWZ36AVANCNFSM6AAAAAAVLNRFCA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Bjorn Heijligers
+31620106733

**Comment by sbarex** [comment]
No, this is not the purpose of the application.

**Comment by heijligers** [comment]
So the plugin is only meant to display additional information for a single
file from within the right-click/context menu?

On Wed, 1 Mar 2023 at 21:14, sbarex ***@***.***> wrote:

> No, this is not the purpose of the application.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/sbarex/MediaInfo/discussions/19#discussioncomment-5170697>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAX7BVZWBQCQFMOCEZTXF5DWZ6U4DANCNFSM6AAAAAAVLNRFCA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Bjorn Heijligers
+31620106733

**Comment by sbarex** [comment]
yes, right.
